<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SQLCheck</class>
 <widget class="QDialog" name="SQLCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SQL Check</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/share/machinetrans/resources/logo.png</normaloff>../../../../usr/share/machinetrans/resources/logo.png</iconset>
  </property>
  <widget class="QTextEdit" name="sqlEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>780</width>
     <height>460</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SQL Commit Data Check</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBttn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>530</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptBttn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>530</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpBttn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>helpBttn</tabstop>
  <tabstop>sqlEdit</tabstop>
  <tabstop>cancelBttn</tabstop>
  <tabstop>acceptBttn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
